src/olympia/devhub/tests/test_views_validation.py

```diff
@@ -63,6 +63,16 @@ def test_upload_processed_validation(self):
         upload = FileUpload.objects.get(uuid=uuid)
         assert upload.processed_validation['errors'] == 1
 
+    def test_login_required(self):
```

Comment:
fair enough, I think I wanted to delete that comment once I saw more of the code :D